imdown-mobile/events/event_detailed.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { FlatList, Image, Linking, Text, TouchableOpacity, View } from "react-native";
+import { EventDetailedStyles } from "./event_detailed_styles";
+import moment from 'moment';
+import Divider  from '../components/divider/divider'
+
+const DOWN_EMOJI_HEIGHT = 82
+const DOWN_EMOJI_WIDTH = 85
+const EVENT_PIC_HEIGHT = 130
+const EVENT_PIC_WIDTH = 130
+const ROW_BUTTON_HEIGHT = 40
+const ROW_BUTTON_WIDTH = 40
+
+const EventDetailed = (props) => {
+const [event, setEvent] = useState(props.route.params.event)
+
+	const renderRSVPUser = ({ item }: { item: String }) => {
+    return (
+      <View>
+            <Text style={EventDetailedStyles.rsvp_user}>{item}</Text>
+      </View>
+    );
+	};
+	
+  return (
+    <View style={EventDetailedStyles.container}>
+				{/* Event title + pic + details box */}
+        <View style={EventDetailedStyles.event_pic_and_title_container}>
+            { event.image_url ?  
+                <View style={EventDetailedStyles.event_picture}>
+                    <Image source = {{uri: event.image_url}}
+                    style = {{ width: EVENT_PIC_WIDTH, height: EVENT_PIC_HEIGHT, borderRadius: 15 }}
```

Comment:
I think inline styles are fine but just curious why you did inline styles here instead of putting it in the stylesheet

Reply:
Yeah I don't have any definitive guideline for what the use inline, and what to put in stylesheet, but I thought that images in general could just use inline styles, as I didn't feel that there were too many possible style parameters that would be used for images, and the style parameters used are rather consistent (dimension and border).  whereas containers and text, there are more possible style parameters that could be used, and style parameters used are more variable (align, justify, color, font, font size, flex)

Reply:
Cool yeah that makes sense